apps/datahub/src/index.html

```diff
@@ -25,7 +25,7 @@
     -->
     <!--%PRELOAD_LINKS%-->
   </head>
-  <body>
+  <body class="overflow-scroll">
```

Comment:
Not tested but I think this should go on `html`, otherwise this won't work well on mobiles (e.g. the address bar won't go away when scrolling)